src/cephadm/cephadm

```diff
@@ -6519,6 +6519,16 @@ def command_rm_cluster(ctx):
                 if os.path.exists(files[n]):
                     os.remove(files[n])
 
+            for fname in glob('/var/log/ceph/cephadm.log*'):
+                os.remove(fname)
```

Comment:
This log is shared between instance of a multi-cluster setup ...

Reply:
To add to this, we do already have a similar situation with the logrotate file and we have a check beforehand if this is the last cluster before we remove it (https://github.com/ceph/ceph/blob/master/src/cephadm/cephadm#L6513-L6515) so maybe we could do something similar here.

Also, this should definitely not be removed if the user provided `--keep-logs`

Reply:
I moved this code to a new section protected by both keep-logs flags and a check to make sure we are the last cluster